src/plugins/autoflow/AutoflowTabularRowController.js

```diff
@@ -68,7 +68,12 @@ define([], function () {
       this.updateRowData.bind(this)
     );
 
-    this.openmct.telemetry.request(this.domainObject, { size: 1 }).then(
+    const options = {
```

Comment:
Should we refactor this file into an ES6 class while we're here?

Reply:
Fair point!

Reply:
I've created a task to move files to ES6 classes here: #6975